src/carousel/carousel.ts

```diff
@@ -39,8 +39,8 @@ export class NgbSlide {
     'class': 'carousel slide',
     '[style.display]': '"block"',
     'tabIndex': '0',
-    '(mouseenter)': 'pause()',
-    '(mouseleave)': 'cycle()',
+    '(mouseenter)': 'mouseEventPause()',
```

Comment:
I would rename it to `onMouseEnter()` as for now it won't be related to pause all the time.